### DISCUSSION TITLE
Unable to connect - "at McTools.Xrm.ConnectionDetail..."

### DISCUSSION THREAD

**Discussion by treesloth** [MscrmTools/XrmToolBox#1074] [Q&A]
Hi.  This is a repeat of an issue posted at MscrmTools.Xrm.Connection.  I think this may be the better place to ask, as this is as likely to be a user problem as something requiring developer help.

So, I'm trying to resolve a failure to connect error.  On my own system I have no problem connecting,  but when I try to help a coworker, they get consistent errors.  I've pasted a screenshot of the error presented in the main UI, and will include the tail of the log file.

My guess, given that the log file seems to jump from discovery to failure, is that the <our org> was not found among the 47 orgs.  Perhaps it's something else.  I suppose it's reasonable-- I'm working from the US, they're in India, so a geographically distinct discovery and entry point server is probable.  Maybe that server is not in sync?  I'm not sure.  FWIW, 4 different systems used by 2 different people have this problem.

Any thoughts or suggestions?  I'm pretty new to this tool and to Dynamics generally, so any pointers that use small words would be immensely appreciated.

```
Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  QueryGlobalDiscovery - Discovery Server Get Orgs Call Complete - Elapsed:00:00:01.6178865
Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  Found 47 Org(s)
Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  Looking for Organization = <our org....> in the results from CRM's Discovery server list.
Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Error	2	10/4/2021 5:30:17 PM  Unable to Login to Dynamics CRM
Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Error	2	10/4/2021 5:30:17 PM  Unable to Login to Dynamics CRM
Source	: Not Provided
Method	: Not Provided
Date	: 10/4/2021
Time	: 5:30:17 PM
Error	: Unable to Login to Dynamics CRM
Stack Trace	: Not Provided
```

![XrmConnectionError](https://user-images.githubusercontent.com/7399050/135897003-2eaa7c83-7c88-4fca-be86-bd9eee6b44a5.jpg)

**Comment by GigiFvanB** [comment]
> Hi. This is a repeat of an issue posted at MscrmTools.Xrm.Connection. I think this may be the better place to ask, as this is as likely to be a user problem as something requiring developer help.
> 
> So, I'm trying to resolve a failure to connect error. On my own system I have no problem connecting, but when I try to help a coworker, they get consistent errors. I've pasted a screenshot of the error presented in the main UI, and will include the tail of the log file.
> 
> My guess, given that the log file seems to jump from discovery to failure, is that the was not found among the 47 orgs. Perhaps it's something else. I suppose it's reasonable-- I'm working from the US, they're in India, so a geographically distinct discovery and entry point server is probable. Maybe that server is not in sync? I'm not sure. FWIW, 4 different systems used by 2 different people have this problem.
> 
> Any thoughts or suggestions? I'm pretty new to this tool and to Dynamics generally, so any pointers that use small words would be immensely appreciated.
> 
> ```
> Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  QueryGlobalDiscovery - Discovery Server Get Orgs Call Complete - Elapsed:00:00:01.6178865
> Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  Found 47 Org(s)
> Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Information	8	10/4/2021 5:30:17 PM  Looking for Organization = <our org....> in the results from CRM's Discovery server list.
> Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Error	2	10/4/2021 5:30:17 PM  Unable to Login to Dynamics CRM
> Microsoft.Xrm.Tooling.Connector.CrmServiceClient	Error	2	10/4/2021 5:30:17 PM  Unable to Login to Dynamics CRM
> Source	: Not Provided
> Method	: Not Provided
> Date	: 10/4/2021
> Time	: 5:30:17 PM
> Error	: Unable to Login to Dynamics CRM
> Stack Trace	: Not Provided
> ```
> 
> ![XrmConnectionError](https://user-images.githubusercontent.com/7399050/135897003-2eaa7c83-7c88-4fca-be86-bd9eee6b44a5.jpg)


Hi treesloth, were you able to find a solution for this? I'm having the same issue
